README.md

```diff
@@ -1,12 +1,4 @@
-![Streisand Logo](https://raw.githubusercontent.com/jlund/streisand/master/logo.jpg "Automate the effect")
-
-- - -
-[English](README.md), [Français](README-fr.md), | [Mirror](https://area51.threeletter.agency)
-- - -
-
-[![Build Status](https://travis-ci.org/jlund/streisand.svg?branch=master)](https://travis-ci.org/jlund/streisand)
-
-
+![Streisand Logo](https://raw.githubusercontent.com/jlund/streisand/master/logo.jpg "Automate the effect") - - - [English](README.md), [Français](README-fr.md), | [Mirror](https://area51.threeletter.agency) - - - [![Build Status](https://travis-ci.org/jlund/streisand.svg?branch=master)](https://travis-ci.org/jlund/streisand) 
```

Comment:
Collapsing this to a single line breaks the layout [when GitHub renders the file](https://github.com/cpu/streisand/blob/e6154e6d05978be88b8e32ea2991f0dcaabad144/README.md).